### DISCUSSION TITLE
How do you customise keymaps with which-key?

### DISCUSSION THREAD

**Discussion by stefanocoretta** [R-nvim/R.nvim#119] [Q&A]
Hi! I wonder if you could help. I am new to Nvim and still not quite sure how to do certain stuff.

I want to set my own keybindings, so I set `user_maps_only = true` and that has worked, but now I can't figure out how to set keybindings with which-key.

I tried the following but it doesn't work.

```
wk.register {
  ['<leader>cr'] = { '<cmd>lua require('r.run').start_R('R')<cr>', 'New [r] terminal' }
}
```

**Comment by PMassicotte** [comment]
This should work just fine if you define it in `ftplugin/r.lua`

```lua
local wk = require("which-key")
wk.register({
    ["<leader>cr"] = { "<cmd>lua require('r.run').start_R('R')<cr>", "New [r] terminal" },
})
```

**Reply by stefanocoretta** [reply]
This doesn't work, but then I realised that my code above has a syntax error (of course I have to use `"` for the command since the command includes `'` 😖, my bad!)
